package.json

```diff
@@ -51,71 +51,39 @@
       "git add"
     ]
   },
-  "jest": {
```

Comment:
Moved jest config to its own file so that I could do `ts-jest config:migrate jest.config.js > new.jest.config.js`